### PR TITLE
sort

### PR DESCRIPTION
**Pull request by hieupc-0248**
## Related Tickets:
- [#69304](https://edu-redmine.sun-asterisk.vn/issues/69304)

## WHAT this PR does?
- thêm tính năng sort

## Notes Impacted Areas:
- resources/views
- resources/lang
- app/Http/Controllers

## Attachments:
- 
image

### CODE REVIEW

**Comment by hieupc-0248** [comment]
ready

**Comment by dungpt-0371** [comment]
oke

**Comment by anhcnn-0378** [comment]
Oke

**Comment by hieupc-0248** [comment]
ready

File: resources/views/welcome.blade.php [hunk 1]
```diff
@@ -11,9 +11,23 @@
...
+            </select>
         <button type="submit" class="p-2 bg-blue-500 text-white rounded-lg" id="searchButton">{{ __('Search') }}</button>
     </form>
 
+    <div class="flex justify-between mx-72 mb-4">
+        <div>
+         
+        </div>
+    </div>
+
```
Comment (ghost) [inline resources/views/welcome.blade.php]: empty?

File: resources/views/welcome.blade.php [hunk 2]
```diff
@@ -25,6 +39,7 @@
                 <div class="flex flex-col justify-between text-center">
                     <div class="flex flex-col justify-between h-24">
                         <a href="books/{{$book->id}}" class="font-bold text-xl mb-2">{{ $book->name }}</a>
+                        <p class="text-gray-700 text-base">Rating: {{ number_format($book->average_rating, 1) }}</p>
```
Comment (ghost) [inline resources/views/welcome.blade.php]: localization

**Comment by hieupc-0248** [comment]
ready

**Pull request merged**